rcgen/src/ring_like.rs

```diff
@@ -22,7 +22,7 @@ pub(crate) fn ecdsa_from_pkcs8(
 
 	#[cfg(all(not(feature = "ring"), feature = "aws_lc_rs"))]
 	{
-		Ok(signature::EcdsaKeyPair::from_pkcs8(alg, pkcs8)._err()?)
+		Ok(signature::EcdsaKeyPair::from_private_key_der(alg, pkcs8)._err()?)
```

Comment:
I don't think we should add a function with whose semantics we can't clearly document because we don't understand them ourselves, so I think you should figure out what the exact format is that OpenSSL generated here, and add API for that -- ideally we'd get aws-lc-rs to more explicitly document what their function support, too.

Reply:
We already support PKCS#8 throughout, but I think it would be okay to add a function that takes a `PrivateSec1KeyDer` and funnels that into this aws-lc-rs API.